src/lib/transaction_logic/zkapp_command_logic.ml

```diff
@@ -975,12 +975,12 @@ module Make (Inputs : Inputs_intf) = struct
     in
     { account_update; account_update_forest; new_frame; new_call_stack }
 
-  let update_sequence_state (sequence_state : _ Pickles_types.Vector.t)
-      sequence_events ~txn_global_slot ~last_sequence_slot =
+  let update_sequence_state (sequence_state : _ Pickles_types.Vector.t) actions
+      ~txn_global_slot ~last_sequence_slot =
     (* Push events to s1. *)
     let [ s1'; s2'; s3'; s4'; s5' ] = sequence_state in
-    let is_empty = Sequence_events.is_empty sequence_events in
-    let s1_updated = Sequence_events.push_events s1' sequence_events in
+    let is_empty = Sequence_events.is_empty actions in
```

Comment:
Do you want to rename the `Sequence_events` module to `Actions` (or maybe `Zkapp_actions`?

Reply:
shoot missed this one